components/statuslight/index.css

```diff
@@ -11,243 +11,258 @@ governing permissions and limitations under the License.
-
-  --spectrum-statuslight-font-size: var(--spectrum-font-size-100);
-
-  /* Space */
-  --spectrum-statuslight-spacing-dot-to-label: var(--spectrum-text-to-visual-100);
-  --spectrum-statuslight-spacing-top-to-dot: var(--spectrum-status-light-top-to-dot-medium);
-  --spectrum-statuslight-spacing-top-to-label: var(--spectrum-component-top-to-text-100);
-  --spectrum-statuslight-spacing-bottom-to-label: var(--spectrum-component-bottom-to-text-100);
-
-  /* Color */
-  --spectrum-statuslight-content-color-default: var(--spectrum-neutral-content-color-default);
-  --spectrum-statuslight-subdued-content-color-default: var(--spectrum-neutral-subdued-content-color-default);
-  --spectrum-statuslight-semantic-neutral-color: var(--spectrum-neutral-visual-color);
-  --spectrum-statuslight-semantic-accent-color: var(--spectrum-accent-visual-color);
-  --spectrum-statuslight-semantic-negative-color: var(--spectrum-negative-visual-color);
-  --spectrum-statuslight-semantic-info-color: var(--spectrum-informative-visual-color);
-  --spectrum-statuslight-semantic-notice-color: var(--spectrum-notice-visual-color);
-  --spectrum-statuslight-semantic-positive-color: var(--spectrum-positive-visual-color);
-  --spectrum-statuslight-nonsemantic-gray-color: var(--spectrum-gray-visual-color);
-  --spectrum-statuslight-nonsemantic-red-color: var(--spectrum-red-visual-color);
-  --spectrum-statuslight-nonsemantic-orange-color: var(--spectrum-orange-visual-color);
-  --spectrum-statuslight-nonsemantic-yellow-color: var(--spectrum-yellow-visual-color);
-  --spectrum-statuslight-nonsemantic-chartreuse-color: var(--spectrum-chartreuse-visual-color);
-  --spectrum-statuslight-nonsemantic-celery-color: var(--spectrum-celery-visual-color);
-  --spectrum-statuslight-nonsemantic-green-color: var(--spectrum-green-visual-color);
-  --spectrum-statuslight-nonsemantic-seafoam-color: var(--spectrum-seafoam-visual-color);
-  --spectrum-statuslight-nonsemantic-cyan-color: var(--spectrum-cyan-visual-color);
-  --spectrum-statuslight-nonsemantic-blue-color: var(--spectrum-blue-visual-color);
-  --spectrum-statuslight-nonsemantic-indigo-color: var(--spectrum-indigo-visual-color);
-  --spectrum-statuslight-nonsemantic-purple-color: var(--spectrum-purple-visual-color);
-  --spectrum-statuslight-nonsemantic-fuchsia-color: var(--spectrum-fuchsia-visual-color);
-  --spectrum-statuslight-nonsemantic-magenta-color: var(--spectrum-magenta-visual-color);
+	/* Static tokens */
+	--spectrum-statuslight-corner-radius: 50%;
+	--spectrum-statuslight-border-width: var(--spectrum-border-width-100);
+
+	--spectrum-statuslight-text-to-visual-75: var(--spectrum-status-light-text-to-visual-75);
+	--spectrum-statuslight-text-to-visual-100: var(--spectrum-status-light-text-to-visual-100);
+	--spectrum-statuslight-text-to-visual-200: var(--spectrum-status-light-text-to-visual-200);
+	--spectrum-statuslight-text-to-visual-300: var(--spectrum-status-light-text-to-visual-300);
```

Comment:
These four additional tokens are not needed. The values used here would be used directly for the value of `--spectrum-statuslight-spacing-dot-to-label` for each t-shirt size.

Reply:
Ok I wasn't sure. I went back and forth on creating these or not, but your way makes it way more streamlined. 

https://github.com/adobe/spectrum-css/pull/2818/commits/6559a91b9c23e272a219f7b908aa494734809729